keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -871,7 +876,7 @@ spec:
   validations:
     - expression: "self.name.startsWith('xyz-')"
       name: name-prefix
-      messageExpression: "self.name + ' must start with xyz-'"
+      messageExpression: "'{} must start with xyz-'.format([self.name])"
```

Comment:
Did you mean to print the field name, or the field value? I believe the field name would be easier for users to understand.

Reply:
Right, that was meant to be field value. I've adjust this/the other examples you mentioned to use field name instead.